README_fr.md

```diff
@@ -0,0 +1,1641 @@
+
+```sh
+(master)$ git rebase -i HEAD~2
+```
+
+Après avoir lancé votre commande rebase, vous verrez quelque chose dans le genre dans votre éditeur de texte :
+
+```vim
+pick a9c8a1d Un peu de refactor
+pick 01b2fd8 Nouvelle feature génial
+pick b729ad5 fixup
+pick e3851e8 autre fix
+
+# Rebase 8074d12..b729ad5 onto 8074d12
+#
+# Commands:
+#  p, pick = use commit
+#  r, reword = use commit, but edit the commit message
+#  e, edit = use commit, but stop for amending
+#  s, squash = use commit, but meld into previous commit
+#  f, fixup = like "squash", but discard this commit's log message
+#  x, exec = run command (the rest of the line) using shell
+#
+# These lines can be re-ordered; they are executed from top to bottom.
+#
+# If you remove a line here THAT COMMIT WILL BE LOST.
+#
+# However, if you remove everything, the rebase will be aborted.
+#
+# Note that empty commits are commented out
+```
+
+Toutes les lignes commençant par un `#` sont des commentaires, elles n'affecteront pas votre rebase.
+
+Vous devez ensuite remplacer les commandes `pick` avec n'importe laquelle parmi celles présentes dans la liste ci-dessus, et vous pouvez également supprimer des commits en supprimant les lignes correspondantes.
+
+Par exemple, si vous voulez **laisser le commit le plus vieux (le premier) tel quel et combiner tous les commits suivants avec le deuxième plus vieux**, vous devez remplacer les lettres à côté de chaque commit sauf le premier et le deuxième par un `f` :
+
+```vim
+pick a9c8a1d Un peu de refactor
+pick 01b2fd8 Nouvelle feature géniale
+f b729ad5 fixup
+f e3851e8 autre fix
+```
+
+Si vous voulez combiner ces commits **et renommer le commit**, il vous faudra ajouter un `r` à côté du deuxième commit ou utiliser simplement `s` au lieu de `f` :
+
+```vim
+pick a9c8a1d Un peu de refactor
+pick 01b2fd8 Nouvelle feature géniale
+s b729ad5 fixup
+s e3851e8 autre fix
+```
+
+Vous pouvez ensuite renommer le commit dans le prochain éditeur de texte qui s'affiche :
+
+```vim
+Nouvelles fonctionnalités encore plus géniales
+
+# Please enter the commit message for your changes. Lines starting
+# with '#' will be ignored, and an empty message aborts the commit.
+# rebase in progress; onto 8074d12
+# You are currently editing a commit while rebasing branch 'master' on '8074d12'.
+#
+# Changes to be committed:
+#   modified:   README.md
+#
+
+```
+
+Si tout se passe bien, vous devriez voir quelque chose comme ceci :
+
+```sh
+(master)$ Successfully rebased and updated refs/heads/master.
+```
+
+#### Stratégie de fusion sûre
+`--no-commit` fait une fusion mais prétend que la fusion a échoué et ne commit par automatiquement, laissant la chance à l'utilisateur d'inspecter plus et de bidouiller plus les résultats de la fusion avant de commiter. `no-ff` garde une preuve qu'une branche de fonctionnalité a auparavant existé, gardant ainsi l'historique du projet consistant :
```

Comment:
```suggestion
`--no-commit` fait une fusion mais prétend que la fusion a échoué et ne commit pas automatiquement, laissant la chance à l'utilisateur d'inspecter plus et de bidouiller plus les résultats de la fusion avant de commiter. `no-ff` garde une preuve qu'une branche de fonctionnalité a auparavant existé, gardant ainsi l'historique du projet consistant :
```
Typo: "par" instead of "pas".